crypto/bls/bls_test.go

```diff
@@ -88,6 +88,13 @@ func TestPublicKey_MarshalText(t *testing.T) {
 	require.EqualError(t, err, fake.Err("couldn't marshal"))
 }
 
+func TestPublicKey_GetPoint(t *testing.T) {
+	point := suite.Point()
+	pk := PublicKey{point: point}
```

Comment:
```suggestion
	pk := NewPublicKeyFromPoint(point)
```

Reply:
Should be the same test as [for the ed25519 implementation](https://github.com/dedis/dela/blob/6cd3d62903806f7863ff29a75c12c6c6bfe06085/crypto/ed25519/ed25519_test.go#L38), in which case the GetPoint() is needed.